petclinic-frontend/src/features/inventories/InventoriesListTable.tsx

```diff
@@ -94,6 +97,39 @@ export default function InventoriesListTable(): JSX.Element {
     }
   };
 
+  useEffect(() => {
+    if (inventoryList.length > 0) {
+      inventoryList.forEach(inventory => {
+        fetchProductQuantity(inventory.inventoryId);
+      });
+    }
+  }, [inventoryList]);
+
+  const fetchProductQuantity = async (inventoryId: string): Promise<void> => {
+    try {
+      const response = await fetch(
+        `http://localhost:8080/api/v2/gateway/inventories/${inventoryId}/productquantity`,
```

Comment:
Maybe you should create another file to handle promises and fetching

Reply:
I find it easier to do it that way but I'll consider changing it later on. Thanks for the feedback!